docs/getting_started.rst

```diff
@@ -14,13 +14,23 @@ Install with Conda
 ------------------
 .. _install-with-conda:
 
-`conda <http://www.continuum.io/blog/conda>`_ is a python package manager built for scientific python. Unlike ``easy_install`` or ``pip``, it handles binaries and binary dependencies, which are critical for most scientific workflows. If you're a ``conda`` user, you can install OpenPathSampling by adding the omnia channel. If you're not a conda user, you should look into it. ::
+We recommend using `conda` to install OpenPathSampling.  `conda
+<http://www.continuum.io/blog/conda>`_ is a python package manager built for
+scientific python. Unlike ``easy_install`` or ``pip``, it handles binaries
+and binary dependencies, which are critical for most scientific workflows.
+If you're a ``conda`` user, you can install OpenPathSampling by adding the
+omnia channel. ::
 
-To install OpenPathSampling with conda, use the following commands ::
+To install the most recent release of OpenPathSampling with conda, use the
+following commands ::
 
   $ conda config --add channels http://conda.binstar.org/omnia
```

Comment:
The travis part is 2-fold: 
- A conda package is build using the `conda-requirements` file. 
- Then a new miniconda environment is created and the previous installed package in installed from _memory_. 

This second installation is used for testing.

Once all tests pass and only PRs on the master branch the docs are built, uploaded to amazon and the new version is uploaded to anaconda.

So we need the requirements to build the package but not afterwards. If this is still the case this should be changed.

Not sure how to have recommended packages. I presume that conda does not support this natively. We could add this to setup.py somehow?

There is also still the question if we want to add the example notebooks to the package or only put them online.